Source/Models/Bookmark.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Bookmark.swift
+//  FolioReaderKit
+//
+//  Created by Omar Albeik on 26.03.2018.
+//
+
+import Foundation
+import RealmSwift
+
+/// A Bookmark object
+open class Bookmark: Object {
+    @objc open dynamic var bookId: String!
```

Comment:
Why are you using `String!`? Could it be optional? If not, you shouldn't use `!`